src/dev-app/datepicker/datepicker-demo.html

```diff
@@ -181,6 +181,8 @@ <h2>Range picker</h2>
       [min]="minDate"
       [max]="maxDate"
       [disabled]="inputDisabled"
+      [comparisonStart]="comparisonStart"
```

Comment:
why not just take in a single `comparisonRange`?

Reply:
It's the same reason why we have the two inputs as separate `ControlValueAccessor`s: I want to avoid requiring people to wrap things in a `DateRange` object before they pass it to Material since it makes consumption difficult with the `forms` module.

Reply:
Ok, that's a good point